app/client/components/productPage.tsx

```diff
@@ -208,41 +201,44 @@ const renderMembershipData = (apiResponse: MembersDataApiResponse) => {
         )}
         <PageContainer>
           {data.regNumber ? (
-            <MembershipRow label={"Membership number"} data={data.regNumber} />
+            <ProductDetailRow
+              label={"Registration number"}
+              data={data.regNumber}
+            />
           ) : (
             undefined
           )}
-          <MembershipRow
-            label={"Membership tier"}
-            data={
-              <div css={wrappingContainerCSS}>
-                <div css={{ marginRight: "15px" }}>{data.tier}</div>
-                <a
-                  href={
-                    "https://membership." +
-                    window.guardian.domain +
-                    "/tier/change"
-                  }
-                >
-                  <Button text="Change tier" />
-                </a>
-              </div>
-            }
-          />
-          <MembershipRow
+          {productType.tierRowLabel ? (
```

Comment:
I've just realised, that we can hide this button if the tier is Patron!

Reply:
i did this, but just realised having the change tier button hidden for patron will block the journey to cancellation (via membership site) (as anti-UX as it is) so I will keep the 'Change Tier' for Patron and add a TODO to revisit once we provide a more direct (but still subtle) path to cancellation